reaktive-testing/src/commonMain/kotlin/com/badoo/reaktive/test/completable/TestAwait.kt

```diff
@@ -0,0 +1,32 @@
+package com.badoo.reaktive.test.completable
+
+import com.badoo.reaktive.completable.Completable
+import com.badoo.reaktive.completable.blockingAwait
+
+/**
+ * This method is used when you need to wait for an asynchronous operation to finish
+ * and you can't use or don't want to use the [TestScheduler][com.badoo.reaktive.test.scheduler.TestScheduler].
+ *
+ * It is JavaScript friendly, it returns `Promise` which is when returned from a test method
```

Comment:
"...it returns `Promise` which, when returned from a test method, causes..."

Reply:
Would you mind to recheck spelling one more time? And the code as well.